app/src/app/[lng]/[inventory]/preferences/transportation/TRANSPORTATION_ITEMS.tsx

```diff
@@ -0,0 +1,23 @@
+export const TRANSPORTATION_ITEMS = [
+  {
+    id: "passenger",
+    icon: "/assets/preferences/transportation/passenger.svg",
```

Comment:
Might be good to shorten these paths as explained on the waste file. And perhaps move all of them into the same file?